src/components/PetInfoSection.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHeadCell,
+  TableRow
+} from "@baltimorecounty/dotgov-components";
+
+const PetInfoSection = props => {
+  const { attributes } = props;
+  const PetAttributeUnits = attributes.filter(
+    item => item.label === "Age Unit" || item.label === "Weight Unit"
+  );
+
+  const PetAttributeRows = attributes
+    .filter(
+      item =>
+        item.label === "Breed" ||
+        item.label === "Color" ||
+        item.label === "Weight" ||
+        item.label === "Age" ||
+        item.label === "Spayed/Neutered" ||
+        item.label === "Shelter Arrival Date"
+    )
+    .map((item, index) => (
+      <TableRow key={index}>
+        <TableHeadCell>{item.label}</TableHeadCell>
+
+        {item.label === "Age" ? (
+          <TableCell>
+            {item.value}{" "}
+            {
+              PetAttributeUnits.filter(item2 => item2.label === "Age Unit")[0]
+                .value
+            }
+          </TableCell>
+        ) : item.label === "Weight" ? (
+          <TableCell>
+            {item.value}{" "}
+            {
+              PetAttributeUnits.filter(
```

Comment:
i think it would make more sense to do something like 

```js
attributes.find(x => x.label === "Weight Unit").value
```

that way you don't have to worry about where the information happens to be in the PetAttributes array.